spec/account/account-manager.spec.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+describe("AccountManager", () => {
+    var accountManager, otDateMock;
+
+    beforeEach(() => {
+        var AccountManager, accountServiceFake, config, hotpFake, promiseMock, randomMock;
+
+        AccountManager = require("../../lib/account/account-manager");
+        otDateMock = require("../mock/ot-date-mock");
+        promiseMock = require("../mock/promise-mock");
+        randomMock = require("../mock/random-mock");
+        accountServiceFake = jasmine.createSpyObj("accountServiceFake", [
+            "completeAsync",
+            "getAsync",
+            "initiateAsync"
+        ]);
+        accountServiceFake.completeAsync = jasmine.createSpy("complete").andCallFake(() => {
+            return promiseMock.resolve(true);
+        });
+        accountServiceFake.getAsync = jasmine.createSpy("get").andCallFake(() => {
+            return promiseMock.resolve({
+                mfaKey: "339r93939303093"
+            });
+        });
+        accountServiceFake.initiateAsync = jasmine.createSpy("initiate").andCallFake((accountId, accountInfo, options) => {
+            return promiseMock.resolve({
+                accountId: accountInfo.accountId,
+                email: accountInfo.email,
+                mfaKey: accountInfo.mfaKey,
+                salt: accountInfo.salt
+            });
+        });
+        hotpFake = jasmine.createSpyObj("hotpFake", [
+            "generateSecretAsync",
+            "verifyToken"
+        ]);
+        hotpFake.generateSecretAsync = jasmine.createSpy("generateSecretAsync").andCallFake(() => {
+            return promiseMock.resolve("thisisasecrectcodefrommfa");
+        });
+        hotpFake.verifyToken = jasmine.createSpy("verifyToken").andCallFake((key, token, opts) => {
+            if (token === "987654") {
```

Comment:
Should be `return token !== "987654"` as I want the opposite.  Just the way I've constructed the tests using the verify token method.